core/templates/pages/exploration-editor-page/translation-tab/services/translation-language.service.ts

```diff
@@ -56,7 +58,7 @@ export class TranslationLanguageService {
     this._activeLanguageChangedEventEmitter.emit();
   }
 
-  getActiveLanguageDescription(): string {
+  getActiveLanguageDescription(): string | null {
```

Comment:
Is there a way to get rid of this `null` here?

Reply:
I don't think we should, because the type comes from `getAudioLanguageDescription` which returns `string | null`. Here the type exists as `string | null` because there might exist a language which have no description, so returning null for those cases.

Reply:
Okay, seems good, could you add comment about this?

Reply:
done